spec/index.bs

```diff
@@ -2653,6 +2676,41 @@ dictionary GPUFenceDescriptor : GPUObjectDescriptorBase {
 </script>
 
 
+Queries {#queries}
+================
+
+## QuerySet ## {#queryset}
+
+<script type=idl>
+interface GPUQuerySet {
+    GPUQueryType getQueryType();
+    GPUSize32 getQueryCount();
+
+    void destroy();
```

Comment:
Is there a specific overhead of GPUQuerySets that means we want a destroy() method? We don't usually include them right now (we only have them on buffers and textures).

Reply:
I'm not sure if I understood correctly, QuerySet should be allocated in a continuous memory on GPU like buffers for storing query data, so I think it need the destory() method.

Reply:
I think that makes sense, though BindGroups also live in continuous memory on some backends but we don't put destroy() on them because they're small. QuerySets are probably typically a little larger.